src/containers/about.js

```diff
@@ -11,9 +13,22 @@ class About extends React.Component {
       margin: '4rem',
     };
 
+    let optionsArr = ['What\'s your name?', 'How are you?']
```

Comment:
In general, use const over let. https://mathiasbynens.be/notes/es6-const
tip: a const variable just cannot be reassigned; however, usually arrays and objs are never reassigned anyway